system/churchcore/classes/qqFileUploader.class.php

```diff
@@ -4,7 +4,7 @@
  * Fileuploader
  */
 class qqFileUploader {
-  private $allowedExtensions = array ();
+  private $allowedExtensions = array();
   private $sizeLimit = 10485760;
```

Comment:
First I didn't touch this line, and second I disagree. Hex-Notation is even less readable.
I would instead expand it to something like 10 * 1024 * 1024, a good opcode cacher (PHP 5.6?) will likely optimize this in the bytecode anyway...